contracts/recovery-contracts/RecoveryForwarder.sol

```diff
@@ -0,0 +1,50 @@
+pragma solidity ^0.4.18;
+
+/**
+ * Basic singleSig contract designed to send funds controlled by signer.
+ */
+contract RecoveryForwarder {
+  // Address which can move funds from this contract
+  address public signer;
```

Comment:
can use something like this to get much better gas on reading signer (i.e. dont have to sload signer every time) https://github.com/wighawag/clones-with-immutable-args. Think it also might save a little deployment cost but not sure